app/models/user.rb

```diff
@@ -1,5 +1,15 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
   has_many :comments
-  has_many :posts
   has_many :likes
```

Comment:
- [ ] Hi @jicamargo, since author_id is specify as the foreign key in both `comments `and `likes `table instead of the default `user_id`, it would be very vital to explicitly let  Rails that `author_id `is the foreign key instead of user_id as you have done with the posts table :+1: so Rails will not rely on the default `user_id `
- Please kindly fix the minor issue before merging your PR. You can also share your thoughts with me concerning your reasoning or my suggestions

Reply:
Thank you for your suggestions. I'll made this changes 👍🏽